spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -65,6 +65,19 @@ def paginated_collection(*args)
       end
     end
 
+    context "when specifying :params option" do
+      let(:collection) do
+        posts = 10.times.map{ Post.new }
+        Kaminari.paginate_array(posts).page(1).per(5)
+      end
+
+      let(:pagination) { paginated_collection(collection, param_name: :post_page, params: { anchor: 'here' }) }
```

Comment:
Line is too long. [111/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.